content/en/docs/contribute/review/reviewing-prs.md

```diff
@@ -163,6 +163,48 @@ When reviewing, use the following as a starting point.
 - Do the changes show up in the Netlify preview? Be particularly vigilant about lists, code
   blocks, tables, notes and images.
 
+### Blog
+
+- Early feedback on blog posts is welcome via a Google Doc or HackMD. Please request input early from the [#sig-docs-blog Slack channel](https://kubernetes.slack.com/archives/CJDHVD54J).
+- Before reviewing blog PRs, be familiar with [Submitting blog posts and case studies](/docs/contribute/new-content/blogs-case-studies/).
+- We are willing to mirror any blog article that was published to https://kubernetes.dev/blog/ (the contributor blog) provided that:
+- the mirrored article has the same publication date as the original (it should have the same publication time too, but you can also set a time stamp up to 12 hours later for special cases)
+  - for PRs that arrive the original article was merged to https://kubernetes.dev/, there haven't been 
```

Comment:
```suggestion
  - [for PRs that arrive _after_ the original article was merged to https://kubernetes.dev/], there haven't been
```

Reply:
You could use parens / round brackets instead. In the UK we might more commonly use square brackets, to mean "the sentence also makes sense if you omit this part".

Reply:
Works for me—I hadn't realized there was a difference between the way we use parens/brackets between here and there. Good to know.